engine/src/main/java/org/terasology/world/block/BlockComponent.java

```diff
@@ -12,7 +12,7 @@
  */
 public final class BlockComponent implements Component {
     @Replicate
-    Vector3i position = new Vector3i();
+    protected Vector3i position = new Vector3i();
     @Replicate
     public Block block;
```

Comment:
I'm not changing this yet to not break Omega. We can easily avoid breaking commit here by making the getter available first, then migrating all modules, and only then restricting the visibility.